packages/babel-plugin-transform-optional-catch-binding/src/index.js

```diff
@@ -0,0 +1,18 @@
+import syntaxOptionalCatchBinding from "babel-plugin-syntax-optional-catch-binding";
+
+export default function() {
+  return {
+    inherits: syntaxOptionalCatchBinding,
+
+    visitor: {
+      CatchClause({ scope, node }) {
+        if (node.param.name) {
+          return;
+        } else {
+          const uid = scope.generateUidIdentifier(node.param.name);
```

Comment:
`node.param.name` is presumably always `null` - I think you instead want a string as the argument to `generateUidIdentifier`, like "unused".